### PR TITLE
DSL documentation changes

### PR DESCRIPTION
**Pull request by rafbcampos**
### Description

Update the DSL documentation to fix imports and give an example of accessing the item value within a Template.

### Change Type

- [x] `patch`
- [ ] `minor`
- [ ] `major`

### CODE REVIEW

**Review by hborawski — APPROVED** [review]

File: docs/site/pages/content/dsl.mdx [hunk 1]
```diff
@@ -94,15 +94,25 @@ Similarly, if a single asset is expected but a list of them is found instead, Pl
 
 ### Templates
 
-Templates are included via the `@player-tools/dsl` package. This can be used in any asset slot:
+Templates are included via the `@player-ui/dsl` package. This can be used in any asset slot:
```
Comment (KetanReddy) [inline docs/site/pages/content/dsl.mdx:97]: I think `@player-tools/dsl` is the correct package. https://github.com/player-ui/tools/blob/main/language/dsl/BUILD#L4

File: docs/site/pages/content/dsl.mdx [hunk 2]
```diff
@@ -94,15 +94,25 @@ Similarly, if a single asset is expected but a list of them is found instead, Pl
...
+};
+
+const bindings = makeBindingsForObject(schema);
+
 <Collection>
   <Collection.Values>
-    <Template data={b`foo.bar`}>
-      <Text>Template Value</Text>
+    <Template data={bindings.foo}>
+      <Text>currentItem._index_.bar: {bindings.foo._index_.bar}</Text>
```
Comment (KetanReddy) [inline docs/site/pages/content/dsl.mdx:114]: For clarity I wouldn't include player syntax for the text. Maybe just:

```suggestion
      <Text>The current item is: {bindings.foo._index_.bar}</Text>
```

**Review by KetanReddy — CHANGES_REQUESTED** [review]
The updated examples look good but I think the dsl package path has been updated to an incorrect package name.

File: docs/site/pages/content/dsl.mdx (hunk 1 above)
Comment (rafbcampos) [reply docs/site/pages/content/dsl.mdx:97]: It seems both export `Template`. However, only the the one exported by `@player-ui/dsl` is ok to receive a `MakeBindingRefable`:

Screenshot 2024-01-12 at 8 09 47 AM

Screenshot 2024-01-12 at 8 09 28 AM

**Review by KetanReddy — APPROVED** [review]

**Pull request merged**